ee/tabby-webserver/src/cron/mod.rs

```diff
@@ -21,9 +28,11 @@ pub async fn run_cron(
     auth: Arc<dyn AuthenticationService>,
     job: Arc<dyn JobService>,
     worker: Arc<dyn WorkerService>,
+    repositories: Arc<RepositoryCache>,
     local_port: u16,
 ) {
     let mut jobs = vec![];
+    let (send, mut receive) = broadcast::channel::<SchedulerJobCompleteEvent>(2);
```

Comment:
how about:

1. define a CronEvents struct
```
struct CronEvents {
  scheduler_job_complete: channel
}
```

and listen the event as part of RepositoryCache::new?

Reply:
Yes, I could do that. I was thinking I'd probably make the event an enum if we need to add more types of them, though.